src/current/v25.1/set-up-logical-data-replication.md

```diff
@@ -85,87 +96,148 @@ When you run LDR in [`immediate` mode](#modes), you cannot replicate a table wit
+Use `CREATE LOGICALLY REPLICATED` to create either a unidirectional or bidirectional LDR stream automatically:
+
+- Unidirectional LDR: run the following from the **destination** cluster:
+
+    {% include_cached copy-clipboard.html %}
     ~~~ sql
-    CREATE LOGICAL REPLICATION STREAM FROM TABLES ({database.public.source_table_name_1},{database.public.source_table_name_2},...)  ON 'external://{source_external_connection}' INTO TABLES ({database.public.destination_table_name_1},{database.public.destination_table_name_2},...);
+    CREATE LOGICALLY REPLICATED TABLE {database.public.destination_table_name} FROM TABLE {database.public.source_table_name} ON 'external://source' WITH unidirectional;
     ~~~
 
-    {{site.data.alerts.callout_info}}
-    {% include {{ page.version.version }}/ldr/multiple-tables.md %}
-    {{site.data.alerts.end}}
+- Bidirectional LDR: This statement will first create the LDR jobs for the first stream. You must run it from the **destination** cluster that does not contain the table. Once the offline initial scan completes, the reverse stream will be initialized so that the original destination cluster can send changes to the original source.
 
-    Once LDR has started, an LDR job will start on the destination cluster. You can [pause]({% link {{ page.version.version }}/pause-job.md %}), [resume]({% link {{ page.version.version }}/resume-job.md %}), or [cancel]({% link {{ page.version.version }}/cancel-job.md %}) the LDR job with the job ID. Use [`SHOW LOGICAL REPLICATION JOBS`]({% link {{ page.version.version }}/show-logical-replication-jobs.md %}) to display the LDR job IDs:
+    Run the following from the **destination** cluster (i.e, the cluster that does not have the table currently):
 
     {% include_cached copy-clipboard.html %}
     ~~~ sql
-    SHOW LOGICAL REPLICATION JOBS;
-    ~~~
-    ~~~
-            job_id        | status  |          targets          | replicated_time
-    ----------------------+---------+---------------------------+------------------
-    1012877040439033857   | running | {database.public.table}   | NULL
-    (1 row)
+    CREATE LOGICALLY REPLICATED TABLE {database.public.destination_table_name} FROM TABLE {database.public.source_table_name} ON 'external://source' WITH bidirectional ON 'external://destination';
     ~~~
 
-    If you're setting up bidirectional LDR, both clusters will have a history retention job and an LDR job running.
+You can include multiple tables in the LDR stream for unidirectional or bidirectional setups. Ensure that the table name in the source table list and destination table list are in the same order:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE LOGICALLY REPLICATED TABLE ({database.public.destination_table_name_1}, {database.public.destination_table_name_2}) FROM TABLE ({database.public.source_table_name_1}, {database.public.source_table_name_2}) ON 'external://source' WITH bidirectional ON 'external://destination', label=track_job;
```

Comment:
is it still the case these sets of table names are supposed to be in the same order in the "from" and "to" list?

also, i don't think we said why? "bad things"?